core/pkg/ingress/controller/annotations.go

```diff
@@ -99,6 +101,7 @@ const (
 	secureUpstream = "SecureUpstream"
 	healthCheck    = "HealthCheck"
 	sslPassthrough = "SSLPassthrough"
+	stickySession  = "StickySession"
```

Comment:
Can we model these as different affinity values instead of a single const called stickysessions?